keanu-project/src/main/java/io/improbable/keanu/vertices/VertexId.java

```diff
@@ -24,6 +24,10 @@ public VertexId() {
         idValues[0] = ID_GENERATOR.getAndIncrement();
     }
 
+    public VertexId(long id) {
```

Comment:
Should we expose this publicly if it's just for debug purposes?

Reply:
I think it's useful for cases where you need to access data from a Map keyed by a VertexID (e.g. PartialDerivatives) and didn't have access to the actual VertexId object that lives on the Vertex. That effectively forces you to have the Vertex if you want to access the Map. This would also be needed if we were deserialising a graph with ids (which we don't currently do but yeah).